Website/DesktopModules/Hotcakes/Core/Admin/Parts/CreditCardGateways/PayPal Payments Pro/Edit.ascx

```diff
@@ -4,16 +4,12 @@
 </h1>
 <div class="hcForm">
     <div class="hcFormItemHor">
-        <asp:Label runat="server" resourcekey="Username" CssClass="hcLabel" />
-        <asp:TextBox ID="txtUsername" runat="server" />
+        <asp:Label runat="server" resourcekey="ClientId" CssClass="hcLabel" />
+        <asp:TextBox ID="txtClientId" runat="server" />
     </div>
     <div class="hcFormItemHor">
-        <asp:Label runat="server" resourcekey="Password" CssClass="hcLabel" />
-        <asp:TextBox ID="txtPassword" runat="server" />
-    </div>
-    <div class="hcFormItemHor">
-        <asp:Label runat="server" resourcekey="Signature" CssClass="hcLabel" />
-        <asp:TextBox ID="txtSignature" runat="server" />
+        <asp:Label runat="server" resourcekey="Secret" CssClass="hcLabel" />
```

Comment:
In most places, there is a prefix to help the code inform developers what objects are. In this case, the `Secret` should be `lblSecret`.  This update will also require updating the key in the requisite localization file.

Reply:
Done. Do you want that I modify this on the rest of the fields that were already present?